apps/meteor/client/components/message/Attachments/DefaultAttachment.tsx

```diff
@@ -25,6 +27,8 @@ const applyMarkdownIfRequires = (
 
 const DefaultAttachment: FC<MessageAttachmentDefault> = (attachment) => {
 	const [collapsed, collapse] = useCollapse(!!attachment.collapsed);
+	const rid = useOpenedRoom();
+	const attachmentMessage = useDecryptedMessage({ ...attachment, msg: attachment.text, rid } as IMessage);
```

Comment:
this is supposed to be only the visual component, we should not include "hard logic here", also useOpenedRoom requires a RoomContext, and we want render messages outside room (like message auditing).

why not decrypt the message just after fetch/receive ?